ethportal-peertest/Cargo.toml

```diff
@@ -2,12 +2,17 @@
 name = "ethportal-peertest"
 version = "0.1.0"
 edition = "2021"
-rust-version = "1.62.0"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+rust-version = "1.66.1"
```

Comment:
Do we need this in every sub-crate Cargo file? Haven't tried it, but maybe only include rust-version in the main Cargo.toml will be enough.